media/css/firefox/independent-splash.less

```diff
@@ -4,15 +4,6 @@
 
 @import "../sandstone/lib.less";
 
-@font-face {
```

Comment:
`independent-splash.less` uses font-awesome for more than just the share widget. Maybe best to leave this one intact?